### PR TITLE
Day 1 "Intro to R/RStudio slides"

### PR DESCRIPTION
**Pull request by sjspielman**
This PR adds day 1 introduction to R and RStudio slides as part of #7. I made two small changes from the previous version - 

1. I replaced a file directory image for teaching paths with an image we used to teach paths in RRP. The new image is easier to read I think. (see slides 9 and 14)
2. I no longer say that RStudio is a company, since soon, they will not be (official Posit switch is October, I believe). Instead, RStudio is now just an IDE. (see slide 20)

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
Add to the schedule as well please!

**Comment by sjspielman** [comment]
> Add to the schedule as well please!

Done.

**Comment by jashapiro** [comment]
The final slide no longer is really needed/accurate. Since we are now using the open source version, we don't have multiple sessions. So the top bar actually looks a bit different, with no session menu and no home button. I don't think we need new screenshots everywhere, but you might note while speaking that it is a bit different.

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -13,34 +13,34 @@ A preliminary schedule for the September 2022 Data Lab Virtual Training Workshop
 |-------------|--------------------------------------------|----------------|
 | **Day 1**   | **2022-09-19** <br> _**Introduction to R and the Tidyverse**_                 |
 | 12:00 PM    | Welcome, Introductions and Getting Started  | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
- 1:00 PM     | Introduction to base R | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call)|
+ 1:00 PM      | [Introduction to R Slides (PDF)](../slides/2022-09-19_Intro-R-RStudio.pdf) <br> Introduction to base R | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call)|
```
Comment (jaclyn-taroni) [inline workshop/SCHEDULE.md]: Pretty minor comment, but the slides usually get linked after we state what the session is

**Review by jaclyn-taroni — APPROVED** [review]
One very low stakes comment

**Comment by sjspielman** [comment]
@jashapiro Do you think it's worth updating screenshots here? I'm kind of on the fence there.. I'm thinking workshop-related items are something to talk about during our pair it later, btw, so we can also discuss then.

**Comment by jashapiro** [comment]
> @jashapiro Do you think it's worth updating screenshots here? I'm kind of on the fence there.. I'm thinking workshop-related items are something to talk about during our pair it later, btw, so we can also discuss then.

I don't think we need new screenshots at this point, but I might note it for a future update.

**Comment by sjspielman** [comment]
I removed the final slide as well as the RRP path slides after discussing more with @jashapiro.

**Pull request merged**